browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1529,6 +1529,9 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		if (data.enabled === 'false' || data.enabled === false)
 			$(pushbutton).prop('disabled', true);
 
+		if (data.visible === 'false' || data.visible === false)
+			L.DomUtil.addClass(pushbutton, 'hidden');
```

Comment:
this also will work, but we do that in `postProcess` for all controls for code sharing for common properties.
I think root cause is that we don't execute `postProcess` after running pushbutton handler inside `_buttonBox`.

Reply:
I added fix https://github.com/CollaboraOnline/online/pull/5984